### ISSUE TITLE
Preload content (latest, featured, popular) on app start

### ISSUE THREAD

**Issue by jin** [jin/NUSWhispers-iOS#5]

**Comment by nicholette** [comment]
This is my plan:

**launching + preload:**
whisperRequestManager keeps a dictionary of section: [whisper]
**switching section:**
every time section opens, get from dictionary instead of calling server
**refresh section:**
update dictionary

@jin Is this what we are looking for?

**Comment by jin** [comment]
Yup lgtm! I was thinking of using NSCache though. http://nshipster.com/nscache/

**Comment by nicholette** [comment]
Aahh ok

**Comment by yangshun** [comment]
:+1:
